AirCasting/UIState/UIState.swift

```diff
@@ -0,0 +1,10 @@
+// Created by Lunar on 10/02/2022.
+//
+
+import Foundation
+import CoreLocation
+
+struct UIState {
+    let expandedCard: Bool
```

Comment:
if this struct is called `SessionCardUIState` we can rename it to just be `isExpanded`